src/DonorProfiles/Exceptions/DuplicateTabException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Give\DonorProfiles\Exceptions;
+
+use Exception;
+
+/**
+ * Define a custom exception class
```

Comment:
@henryholtgeerts `@since` tag is missing from most of the classes. Can you add?

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5569/commits/4f6b431d0fead61d30cbae701f7c0ce7b65b94a5